R/prepareOutputForNullModel.R

```diff
@@ -182,11 +203,22 @@
     zeroFLAG <- vc.mod$zeroFLAG
     niter <- vc.mod$niter
 
-    out <- list(family = family, hetResid = hetResid, varComp = varComp,
+    # Sample-level data frame.
+    fit <- data.frame(
+      outcome = as.vector(y),
+      workingY = as.vector(workingY),
+      fitted.values = fitted.values,
+      resid.marginal = resid.marginal,
```

Comment:
@amstilp I think adding in `linear.predictor` might be as simple as including `vc.mod$eta` here

Reply:
Done